1-the-basics/index.html

```diff
@@ -0,0 +1,20 @@
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Daniel Contreras</title>
+  </head>
+  <body>
+      <h1>My first web page</h1>
+      <h2>Hobbies</h2>
+      <ul>
+          <li>Play in Pc</li>
+          <li>Ride bike</li>
+          <li>Learn about develop</li>
+      </ul>
+    <p>Daniel Felipe Contreras López and i am really enjoing this course</p>
+    <img src="https://i.pinimg.com/originals/44/4c/1c/444c1cb1ba24c80caa6c5fb9d7735785.jpg" alt="Marine Terran" style="width: 200px;">
+    <br>
+    <a href="https://github.com">This is my GitHub Page</a>
+  </body>
+</html>     
```

Comment:
Good job, remember to create a new branch for each exercise. Only for today let's keep the basic HTML exercise and semantic exercise in the same Pull Request.

Reply:
Using <br> it's a bad practice because it's not a semantic tag. We are going to see how to put a line break with CSS.